import model.xception.model1_val1 as model1

model1.train()